server/routes/testutils/appSetup.ts

```diff
@@ -31,8 +31,10 @@ const testUserWithEditorRole = {
     'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6InVzZXIxIiwiaWF0IjoxNTE2MjM5MDIyLCJhdXRob3JpdGllcyI6WyJST0xFX0VEVUNBVElPTl9XT1JLX1BMQU5fRURJVE9SIl19.vZ7FwrGzUFIDgxluRAj72It0yBW2hbxu5UQCjKkyOzM',
   username: 'user1',
   displayName: 'First Last',
-  activeCaseLoadId: 'MDI',
-  authSource: 'NOMIS',
+  activeCaseLoadId: 'BXI',
```

Comment:
This sets up the test user that is used in unit tests (not the cypress integration tests) - needed to make this change to set the users caseloads (and also their role wasnt being set before, so I've done that too)